.github/workflows/test-multiple-versions.yml

```diff
@@ -0,0 +1,45 @@
+name: Test Multiple Versions
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
@dai-shi i think [using a matrix](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) can work well here, eg:
```yaml
    strategy:
      matrix:
        reactVersion: ['16.8.0', '17.0.2', '18.0.0-alpha-05726d72c-20210927', '0.0.0-experimental-05726d72c-20210927']
```

and then below
```yml
      - name: Test ${{ matrix.reactVersion }}
        run: |
          yarn add react@${{ matrix.reactVersion }} react-dom@${{ matrix.reactVersion }}
          yarn test:ci
```